core/schema.py

```diff
@@ -1755,22 +1780,26 @@ def mutate_and_get_payload(cls, root, info, username, token, new_password, **inp
 class OpenimisObtainJSONWebToken(mixins.ResolveMixin, JSONWebTokenMutation):
     """Obtain JSON Web Token mutation, with auto-provisioning from tblUsers """
 
+    csrf_token = graphene.String()
+
     @classmethod
     def mutate(cls, root, info, **kwargs):
 
         username = kwargs.get("username")
         password = kwargs.get("password")
         request = info.context
 
-        if CoreConfig.csrf_protect_login:
```

Comment:
This cause problems/issues  when this new approach is used. After logout the csrf token error still appears on login page, causing blocking login into application @delcroip

Reply:
the app should first fetch the CSRF token

Reply:
wherever I add this block, it's still the issue I described at the top of this thread of comments.